typescript/api-client/projects/vcd/sdk/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@vcd/sdk",
+  "version": "0.12.0",
+  "peerDependencies": {
+    "@angular/common": "8.2.14",
+    "@angular/core": "8.2.14",
```

Comment:
Angular peer dependencies need to be a range. For now it will need to support 9.1 through current VCD version, which is 4.4 through 8. Leave this package version at 0.11.0 for now because we'll need to separately merge this into other branches.

Once this is a real Angular project on master, and ported to release branches, we can talk about freezing release branches for specific versions of VCD (like 9.1 and 9.5).

Reply:
I downgraded the peer dependencies to 4.4.4, also tested the plugin on 10.0 and 9.5 env it works along with the i18n package from @vcd/i18n.